File: print_segments.js
const data = require('./src/timestamps.json');
console.log('Total duration:', data.duration, 'seconds, totalFrames:', data.totalFrames);
console.log('Segments count:', data.segments.length);

data.segments.forEach((s, i) => {
  const fStart = Math.round(s.start * 30);
  const fEnd = Math.round(s.end * 30);
  console.log(`[Seg ${i+1}] [${s.start}s - ${s.end}s | F${fStart} - F${fEnd} (${fEnd - fStart}f)]: ${s.text}`);
});

File: print_segments.py
import json

with open('src/timestamps.json', encoding='utf-8') as f:
    d = json.load(f)

print(f"Total Duration: {d['duration']}s | Total Frames: {d['totalFrames']}")
for s in d['segments']:
    sf = int(round(s['start'] * 30))
    ef = int(round(s['end'] * 30))
    print(f"[{s['id']:2d}] {sf:4d} -> {ef:4d} ({s['start']:5.1f}s - {s['end']:5.1f}s): {s['text']}")
